querydsl-core/src/main/java/com/querydsl/core/types/TemplateFactory.java

```diff
@@ -183,9 +182,9 @@ public Template create(String template) {
                 if (m.start() > end) {
                     elements.add(new Template.StaticText(template.substring(end, m.start())));
                 }
-                String premodifiers = m.group(1).toLowerCase(Locale.ENGLISH);
+                String premodifiers = m.group(1).toLowerCase();
                 int index = Integer.parseInt(m.group(2));
-                String postmodifiers = m.group(6).toLowerCase(Locale.ENGLISH);
+                String postmodifiers = m.group(6).toLowerCase();
```

Comment:
Not that we will, but if we would add`i`, that would then fail in a Turkish `Locale`.